src/PKSim.Infrastructure/ProjectConverter/v6_1/Converter602To612.cs

```diff
@@ -127,7 +127,7 @@ private void clearFormulaCacheInReactionBuildingBlock(IReactionBuildingBlock rea
 
       public void Visit(Formulation formulation)
       {
-         if (!formulation.FormulationType.IsOneOf(CoreConstants.Formulation.Weibull, CoreConstants.Formulation.Lint80, CoreConstants.Formulation.Particles, CoreConstants.Formulation.Table))
+         if (!formulation.FormulationType.IsOneOf(CoreConstants.Formulation.Weibull, CoreConstants.Formulation.Lint80, CoreConstants.Formulation.Table))
             return;
```

Comment:
With the new implementation there is no Is_Suspension parameter for particles anymore (it can be only suspension)